skin/frontend/base/default/css/aims/pledg.css

```
@@ -73,4 +74,26 @@
 
 @media only screen and (max-width: 767px) {
     .pledg_payment_methods { width: 100%; margin-bottom: 10px !important; }
+}
+
+/* Payment Schedule */
+.paymentScheduleContainer {
+    margin: 1em 0 0 1em;
+    width: fit-content;
+    position: relative;
+    transition: opacity 0.3s ease-in-out;
+}
+
+.paymentScheduleFees {
+    float: right;
+    margin-top: -7px;
+    margin-bottom: 7px;
+    font-size: small;
+}
+
+.toHide {
+    opacity: 0;
```

L'opacity est assez dangereux dans le sens ou si, pour une raison x ou y, on ne nettoie pas le innerHTML au moment du changement de payment method: 
https://github.com/pledgcorporate/ecard-magento1.9/pull/8/files#diff-1bfc6438248106a8f70c83b0da19d6d0498628a242fd24b823bd50548b32ea60R19

On va se retrouver avec un bon gros carré blanc vide mais cliquable

Le display none ferait mieux le taff.
```suggestion
    display: none;
```

Si on avait jquery on pourrait faire un .show() ou .hide() mais je suppose que c'est lié au theme et non au module.

---

Pas de jquery non, c'est pour ça que j'ai utilisé l'opacity qui permet d'avoir un semblant de fadeIn